modular_skyrat/modules/sec_haul/code/guns/bullets.dm

```diff
@@ -51,48 +51,48 @@
 	projectile_type = /obj/projectile/bullet/advanced/b6mm/ihdf
 	harmful = TRUE
 
-/obj/projectile/bullet/advanced/b6mm/ihdf
-	name = "6.3mm fragmentation pellet"
+/obj/projectile/bullet/advanced/b6mm/frag
+	name = "6mm fragmentation pellet"
 	icon_state = "bullet_i"
-	damage = 15
-	bare_wound_bonus = 50
-	embedding = list(embed_chance=60, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
+	damage = 25
+	bare_wound_bonus = 20
+	embedding = list(embed_chance=40, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
 	weak_against_armour = TRUE
 
 ///////////9x19mm/////////////////
 //FMJ | JHP | IHDF | RUBBER//
 ///////////////////////////////
 /obj/item/ammo_casing/b9mm
-	name = "9x19mm peacekeeper FMJ casing"
+	name = "9mm peacekeeper casing"
 	desc = "A 9x19mm bullet casing with a low powder load to prevent breaches."
```

Comment:
So it has low powder load, but you're buffing it to deal more damage and actually be faster than normal?

Reply:
The way speed works with bullet code is really stupid, which means that the higher the value = the slower it actually is